index.js

```diff
@@ -2,21 +2,111 @@
  * Reverse Polish Notation Calculator
  */
 
-var arguments = process.argv.slice(2);
+let arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+let operator = [];
+let operand = [];
 
-console.log('Calculating:', operand1, operand2, operator);
+for (let k = 2; k < arguments.length; k++) {
+  if (k % 2 == 0) {
+    operator.push(arguments[k]);
+  }
+}
+
+for(let i = 0; i < arguments.length; i++){
+  if (i % 2 == 0){
+
+  }else{
+    operand.push(arguments[i]);
+  }
+
+  if(i==0){
+    operand.push(arguments[i]);
+  }
+}
+
+if(arguments == 0){
+  console.log('Supported operators:\n\r +, -, *, /');
```

Comment:
`\n` is usually sufficient.

```suggestion
  console.log('Supported operators:\n +, -, *, /');
```